layers/+tools/quickurl/packages.el

```diff
@@ -0,0 +1,62 @@
+;;; packages.el --- quickurl layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Spenser Truex <web@spensertruex.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
```

Comment:
Please remove the sample comments here

Reply:
I deleted that whole file. @smile13241324 make sure to rebase your code since I did.